### ISSUE TITLE
[Bug] [NV14] can't exit out of widgets setup

### ISSUE THREAD

**Issue by LimitlessGreen** [EdgeTX/edgetx#579]
Hello,
I'm not able to exit out the *Setup widgets* dialog. This behavior effects only the first tab where the top elements are changeable. Also only the first two boxes should have the ability to put a widget in (otherwise bat, time and RSSI widgets will be overdrawn). 

**To Reproduce**
Touch the screen -> Screen Settings -> First Tab (User Interface) -> Setup widgets

**Expected behavior**
Touch the logo on the top left to exit.

**Screenshot**
![widgets_setup](https://user-images.githubusercontent.com/15064073/128912498-69e237af-06dc-4376-9c46-f7cd56a734ee.jpg)

**In short**
- [ ] Make the exit at "Setup widgets" work
- [ ] Hide the right two widget boxes  

**General information**
 - EdgeTX version (2.5.0 nightly, f5dec4e31edd8181e6a9925c31a03fa2cb659eef)
 - Hardware (FlySky Nirvana NV14)
 
 ---
PS: Are the not selectable themes and the only white background a symptom of "not yet implemented"? :octocat: 

_Thanks for the nice project :)
-- Limitless Green --_

**Comment by LimitlessGreen** [comment]
It's possible to exit out with the exit button, just not with the logo.

**Comment by eshifri** [comment]
I can reproduce it. Will take a look.

**Comment by raphaelcoeffic** [comment]
Fixed by #584
